scaletest/terraform/scenario-medium2x.tfvars

```diff
@@ -0,0 +1,5 @@
+nodepool_machine_type_coder      = "t2d-standard-8"
+nodepool_machine_type_workspaces = "t2d-standard-8"
+nodepool_size_workspaces         = 2
```

Comment:
I don't understand the "2x" sizing.  In the case of large2x, it means 2 coder replicas, but here is means 2 workspace nodes?

Reply:
For now I'm just sticking to small, medium, and large. We can iterate on more scenario specifics later.